sparksql-scalapb/src/main/scala/scalapb/spark/TypeMappers.scala

```diff
@@ -0,0 +1,19 @@
+package scalapb.spark
```

Comment:
Can this file be in sparksql-scalapb/src/test/scala?

Reply:
looks like this file isn't used anywhere. gonna delete it.

Reply:
I feel like this should be in the `src/main` because we have reference in the protobuf message to generate type conversion. But I can see that this type mapper maybe only useful to our specific use case, so we can move to test sources if you think it's better.